app/src/main/kotlin/app/lawnchair/lawnicons/util/GetIconInfo.kt

```diff
@@ -9,7 +9,7 @@ fun Context.getIconInfo(): List<IconInfo> {
     val iconInfo = mutableListOf<IconInfo>()
 
     try {
-        val xmlId = R.xml.grayscale_icon_map
+        val xmlId = R.xml.theme_config
         if (xmlId != 0) {
```

Comment:
we don't need grayscale_icon_map for lawnchair to provide theme icons but to provide backward compaitable with older lawnicons we should keep that in lawnchair.

So used new name _theme_config_.